pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -108,13 +109,20 @@ func generateDefaulterPath(gvk kube_schema.GroupVersionKind) string {
 	return fmt.Sprintf("/default-%s-%s-%s", strings.Replace(gvk.Group, ".", "-", -1), gvk.Version, strings.ToLower(gvk.Kind))
 }
 
-func addValidators(mgr kube_ctrl.Manager) error {
-	handler, err := k8s_webhooks.NewValidatingWebhook(k8s_resources.DefaultConverter(), core_registry.Global(), k8s_registry.Global())
-	if err != nil {
-		return err
-	}
+func addValidators(mgr kube_ctrl.Manager, rt core_runtime.Runtime) error {
+	handler := k8s_webhooks.NewValidatingWebhook(k8s_resources.DefaultConverter(), core_registry.Global(), k8s_registry.Global())
```

Comment:
Let's wrap this handler into a "composite" handler (aggregate over a list of handlers).

 "composite" handler goes over a list and calls `Handle` until the first denied response.

The list of handlers will consist of this one and the new Mesh-specific one.